external/Makefile

```diff
@@ -105,7 +114,9 @@ $(BIN): $(OBJS) iotivity_build
 	$(call ARCHIVE, $@, $(OBJS))
 
 .depend: Makefile $(SRCS)
-	$(Q) $(MKDEP) $(DEPPATH) "$(CC)" -- $(CFLAGS) -- $(SRCS) >Make.dep
+	$(Q) $(MKDEP) $(DEPPATH) "$(CC)" -- $(CFLAGS) -- $(ARCS) >Make.dep
```

Comment:
The followint options in the CFLAGS are not valid for .cpp files.
 -Wstrict-prototypes
 -Wno-implicit-function-declaration